### ISSUE TITLE
Session rotation problem

### ISSUE THREAD

**Issue by nexxpix** [thelia/thelia#1740]
Hi,

I see a small non apprehended problem: the sessions are stored in a defined directory that is "local/session", but this method blocks the file rotation. For a high daily traffic ecommerce, I have got 500MB  of files in 2 months =/

My solution has been to manually create a cron eliminating the session directory every month, example:
0 0 1 * * rm -R /public_html/www/local/session

I think we need to find a lasting solution to this problem...

**Comment by nexxpix** [comment]
Moreover, I think there are a lot of file created in this directory. Maybe there are duplicates files ?

**Comment by driou** [comment]
Hi, 

It seems that is the problem come from your server configuration. If you configure your PHP environment correctly, it must be cleaned frequently. You have to set up this kind of configuration :
http://www.dev-metal.com/how-the-php-session-garbage-collector-really-works/

Afterwords it's well configured, it will also solve the problem of lot of file created in this directory.

**Comment by nexxpix** [comment]
Indeed, our parameter session.gc_probability was initialized to 0 instead of 1.

Maybe add a check during the install, or to force the default value by htaccess, OVH has initialized the parameter to 0 (http://pro.ovh.net/test.php) and he will block the site if the quota is achieved.

Thanks
